Docs/index.md

```diff
@@ -0,0 +1,18 @@
+<img src="https://www.w3.org/Icons/SVG/svg-logo-v.png" alt="W3C SVG Logo">
+
+# SVG.NET Documentation
```

Comment:
I know I wrote this initially, but calling it SVG.NET is not entirely correct, as the library is just called "SVG". May use "SVG Library Documentation" instead.

Reply:
Maybe SVG.NET, if you wish to stress it is a .NET implementation of SVG. Remember when we had Visual Studio .NET?

Reply:
Yes, that was my intention, but it is not really the name of the library. It is already a bit confusing - the name of the organization is shown as "SVG.NET", but written as "svg-net" because of the naming rules for GitHub organizations, there is a copy of the library called "SVG.Core" by somebody, and there is a separate library "SVGNET". 
As it is called just "Svg" in nuget, I think we should stick to that name, though that is just my current opinion.